docs/dev/build_riscv64.md

```diff
@@ -0,0 +1,184 @@
+# Cross compile OpenVINO™ Runtime for RISCV64 systems
+This guide shows how to build OpenVINO Runtime for 64-bit RISC-V devices. Due to limited resources, cross compilation is used now for building OpenVINO targeting RISC-V development boards.
```

Comment:
```suggestion
This guide shows how to build OpenVINO Runtime for 64-bit RISC-V devices using cross-compilation.
```

I'm not sure - should we leave some words about reason of cross-compilation since it's just build tutorial 🤔

Reply:
If we will write another separate doc to introduce the build on RISC-V host (maybe `build_linux.md` is enough 🤔), we do not need to show the reasons here. If all RISC-V related tutorial is contained in this file, maybe the reason is necessary to show the current progress. 🤔